ls1intum/Artemis
# Fix Feedback Editor Alignment Issue

### Checklist

### Motivation and Context


Fixes #3551 

### Description

Fixes alignment issues

### Steps for Testing


1. Log in to Artemis
2. Open a Text Submission to grade
2. Check the Feedback editor upon clicking on a block. 

### Test Coverage




### Screenshots


![final](https://user-images.githubusercontent.com/18335489/121771460-c90fdb00-cb6f-11eb-934c-6930e1e6b8bc.PNG)
